applications/FluidDynamicsApplication/tests/cpp_tests/test.py

```diff
@@ -0,0 +1,30 @@
+
+from KratosMultiphysics import *
```

Comment:
this file seems to be only used for debugging, I don't think it should be committed

Reply:
The file was deleted